torch/csrc/dynamo/eval_frame.c

```diff
@@ -39,16 +39,26 @@ static void eval_frame_callback_set(PyObject* obj) {
 #if IS_PYTHON_3_11_PLUS
 #define THP_EVAL_API_FRAME_OBJECT _PyInterpreterFrame
 
+#else
+// Things specific to PY < 3.11
+#define THP_EVAL_API_FRAME_OBJECT PyFrameObject
+static int
+THP_PyFrame_FastToLocalsWithError(THP_EVAL_API_FRAME_OBJECT *frame, int *free_vars_copied) {
```

Comment:
I deleted this function recently - please remove.

Reply:
Nice, will update.